docs/UserGuide.md

```diff
@@ -5,44 +5,111 @@ title: User Guide
 
 ![gitGud](images/gitgud.png)
 
-gitGud is a **desktop application for managing your friends' gaming information**. It uses a gamer-themed **Graphical User Interface 
-(GUI) combined with an optimized Command Line Interface (CLI)** to give you a smooth and efficient experience. 
+* Table of Contents
+  {:toc}
 
-Keeping track of all of yours friends' gaming information and schedules can be difficult and time-consuming.
-With gitGud, there is no need to get tilted(frustrated) by this anymore as **how fast you type determines how fast you can manage your gaming contacts**.
+--------------------------------------------------------------------------------------------------------------------
+## Introduction
 
+### What is gitGud? 
 
-* Table of Contents
-{:toc}
+gitGud is a **desktop application for storing and managing your friends' gaming information and schedules**. 
+It is dedicated to gamers like you and provides a gamer-themed experience with a focus on increasing your efficiency through
+its typing-focused interface.  
```

Comment:
'typing-focused' vs 'typing-based'
The word 'focus' has already been used in the sentence, feels a bit repetitive to use it again.